app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
-    drafts,
-    caption="Draft services",
-    empty_message="You haven’t added any services yet." if framework.status == 'open' else "You didn’t add any services.",
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(draft.serviceName,
-                              url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)) }}
+  {% set ns = namespace(draft_rows = []) %}
+  {% for draft in drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+    {{ govukButton({
+      "text": "Make a copy",
+      "classes": "govuk-!-margin-0",
+    }) }}
+    </form>
+    {% endset %}
 
-      {% if framework.status == 'open' %}
-        {{ summary.text(submission.multiline_string(
-          submission.can_be_completed_text(draft.unanswered_required, framework.status),
+    {% set row = ns.draft_rows.append(
+      [
+          {"html": service_link},
+          {"html": submission.multiline_string(submission.can_be_completed_text(draft.unanswered_required, framework.status),
           submission.unanswered_required_text(draft.unanswered_required, draft.unanswered_optional),
-          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional)
-        )) }}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
+          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional))},
+          {"html": make_copy_button},
+      ]
+    ) %}
+{% endfor %}
+<p class="summary-item-top-level-action">
+  <a href="{{ url_for(".start_new_draft_service", framework_slug=framework.slug, lot_slug=lot.slug) }}" class="summary-change-link">Add a service</a>
+</p>
 
+    {% if not drafts %}
+      {{"You haven’t added any services yet." if framework.status == 'open' else "You didn’t add any services."}}
+    {% endif %}
```

Comment:
This text used to appear in the table, I've moved it to plain text above the table. 

## Before 
![Screenshot 2021-06-17 at 15 52 01](https://user-images.githubusercontent.com/1764158/122420984-f996b100-cf83-11eb-9026-fd2b8626dcba.png)

## After
![Screenshot 2021-06-17 at 15 52 36](https://user-images.githubusercontent.com/1764158/122421106-116e3500-cf84-11eb-8979-84b06ab620fd.png)

Reply:
I think we should jettison the table in this case as well - just keep the plain text message, since there's no point displaying an empty table.

Reply:
fixed as suggested in ff4b27d